addons/github/models.py

```diff
@@ -185,6 +185,27 @@ def is_private(self):
         except GitHubError:
             return
 
+    def get_folders(self, **kwargs):
+        if self.has_auth:
```

Comment:
Can you return something if `self.has_auth` is False?  I agree you should have this check to future-proof this method, even though right now looks like this method cannot be called unless the user is a github addon authorizer in both v1 and v2.

Reply:
✅ Done